plugins/libplugin.c

```diff
@@ -33,10 +33,13 @@ struct plugin_conn {
 	MEMBUF(char) mb;
 };
 
+/* Connection to make RPC requests. */
+static struct plugin_conn rpc_conn;
```

Comment:
Whaaaaat?  I suggest retaining in `command`, maybe require that objects keep a reference to this.  This commit by itself shows no reason to make this global.

Reply:
The next commit needs it, however.